docs/resource/host.md

```diff
@@ -16,6 +16,12 @@
 | `host.image.name` | string | Name of the VM image or OS install the host was instantiated from. | `infra-ami-eks-worker-node-7d4ec78312`; `CentOS-8-x86_64-1905` | Recommended |
 | `host.image.id` | string | VM image ID or host OS image ID. For Cloud, this value is from the provider. | `ami-07b06b442921831e5` | Recommended |
 | `host.image.version` | string | The version string of the VM image or host OS as defined in [Version Attributes](README.md#version-attributes). | `0.1` | Recommended |
+| `host.ipv4.addresses` | string[] | Available IPv4 addresses of the host, excluding loopback interfaces. [1] | `[['192.168.1.140']]` | Opt-In |
+| `host.ipv6.addresses` | string[] | Available IPv6 addresses of the host, excluding loopback interfaces. [2] | `[['fe80::abc2:4a28:737a:609e', 'fe80::849e:eaff:fe31:3c90']]` | Opt-In |
+
+**[1]:** Addresses MUST be specified in dotted-quad notation.
```

Comment:
Quoting [Wikipedia](https://en.wikipedia.org/wiki/Dot-decimal_notation),

> No formal specification of this textual IP address representation exists

The Wikipedia article also links [this expired draft](https://datatracker.ietf.org/doc/html/draft-main-ipaddr-text-rep-02).